setup.py

```diff
@@ -92,13 +95,16 @@ def finalize_options(self):
       ext_modules = [module1, module2, module3],
```

Comment:
Can you increase the version to the next version, say 0.2.1.2?

Reply:
I'm not sure it is necessary to make a new patch release, as no real changes to the library itself were made. The binaries should be identical to if you were to install this yourself (or at least that was the goal). If you want to make a new patch release, I gues you can do so, if you'd like, but I don't see it is as a necessity for this PR. But up to you :]